backend/homework/9/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.profiles.active=production
```

Comment:
what is the point of using profiles if we have same configurations for both, atleaast for demo add some configs